sfepy/mechanics/tensors.py

```diff
@@ -96,6 +96,28 @@ def get_deviator(tensor, sym_storage=True):
 
     return dev
 
+
+def get_cauchy_strain(grad):
+    nc, _, dim = grad.shape
```

Comment:
"""
Given a gradient, return the corresponding Cauchy strain (symmetric gradient).
"""
- add simple docstring?

Reply:
OK